packages/sw-background-sync-queue/src/lib/queue-utils.js

```diff
@@ -33,6 +33,12 @@ async function getQueueableRequest({request, config}) {
 	return requestObject;
 }
 
+/**
+ * takes JSON object and return a Request object to be executed by
```

Comment:
Couldn't you say "Takes and object" rather than "takes a JSON object".

The input is parsed as JSON or anything.